app/pathogen/arbovirus/wip/visualizations-section.tsx

```diff
@@ -0,0 +1,56 @@
+import { useCallback } from "react";
+import { useRouter } from "next/navigation";
+import { VisualizationId, VisualizationInformation, addToVisualizationInformation } from "../visualizations/visualizations";
+import { ZoomIn } from "lucide-react";
+import { cn } from '@/lib/utils';
+
+export const VisualizationsSection = () => {
+    const router = useRouter();
+    const allVisualizationInformationWithClassnames = addToVisualizationInformation({
+      [VisualizationId.MEDIAN_SEROPREVALENCE_BY_WHO_REGION_AND_AGE_GROUP]: { className: "h-full" },
+      [VisualizationId.ESTIMATE_COUNT_BY_WHO_REGION_AND_ARBOVIRUS]: { className: "h-1/2" },
+      [VisualizationId.ESTIMATE_COUNT_BY_ARBOVIRUS_AND_ANTIBODY_TYPE ]: { className: "h-1/2" },
+      [VisualizationId.MEDIAN_SEROPREVALENCE_BY_WHO_REGION]: { className: "h-full" },
+      [VisualizationId.CUMULATIVE_ESTIMATE_COUNT_OVER_TIME_BY_ARBOVIRUS]: { className: "h-1/2" },
+      [VisualizationId.CUMULATIVE_ESTIMATE_COUNT_OVER_TIME_BY_SAMPLE_FRAME]: { className: "h-full 2xl:h-3/4" },
+      [VisualizationId.TOP_TEN_COUNTRIES_REPORTING_ESTIMATES_BY_ARBOVIRUS]: { className: "h-full" },
+    })
+
+    const visualizationsOnLeftSide = allVisualizationInformationWithClassnames.filter((visualizationInfo) => [
+      VisualizationId.ESTIMATE_COUNT_BY_WHO_REGION_AND_ARBOVIRUS,
+      VisualizationId.ESTIMATE_COUNT_BY_ARBOVIRUS_AND_ANTIBODY_TYPE,
+      VisualizationId.CUMULATIVE_ESTIMATE_COUNT_OVER_TIME_BY_SAMPLE_FRAME
+    ].includes(visualizationInfo.id));
+    const visualizationsOnRightSide = allVisualizationInformationWithClassnames.filter((visualizationInfo) => [
+      VisualizationId.MEDIAN_SEROPREVALENCE_BY_WHO_REGION,
+      VisualizationId.MEDIAN_SEROPREVALENCE_BY_WHO_REGION_AND_AGE_GROUP
+    ].includes(visualizationInfo.id));
```

Comment:
Intentionally not adding all of the visualizations, Harriet wanted some removed so I'm just including a subset.